feature-libs/organization/account-summary/components/account-summary/details/document/account-summary-document.component.html

```diff
@@ -0,0 +1,192 @@
+                  {{ 'orgAccountSummary.document.date' | cxTranslate }}
+                </div>
+                <div class="cx-account-summary-document-value">
+                  {{ document?.date | cxDate: 'longDate' }}
+                </div>
+              </td>
+              <td class="cx-account-summary-document-date">
+                <div class="cx-account-summary-document-label">
+                  {{ 'orgAccountSummary.document.dueDate' | cxTranslate }}
+                </div>
+                <div class="cx-account-summary-document-value">
+                  {{ document?.dueDate | cxDate: 'longDate' }}
+                </div>
+              </td>
+              <td class="cx-account-summary-document-monetary">
+                <div class="cx-account-summary-document-label">
+                  {{ 'orgAccountSummary.document.originalAmount' | cxTranslate }}
+                </div>
+                <div class="cx-account-summary-document-value">
+                  {{ formatCurrency(document.amount, document.currency) }}
+                </div>
+              </td>
+              <td class="cx-account-summary-document-monetary">
+                <div class="cx-account-summary-document-label">
+                  {{ 'orgAccountSummary.document.openAmount' | cxTranslate }}
+                </div>
+                <div class="cx-account-summary-document-value">
+                  {{ formatCurrency(document.openAmount, document.currency) }}
+                </div>
+              </td>
+              <td class="cx-account-summary-document-status">
+                <div class="cx-account-summary-document-label">
+                  {{ 'orgAccountSummary.document.status' | cxTranslate }}
+                </div>
+                <div class="cx-account-summary-document-value">
+                  <span [ngClass]="{'status-open': document.status === 'open', 'status-closed': document.status === 'closed'}">
+                    {{ document?.status }}
+                  </span>
+                </div>
+              </td>
+              <td class="cx-account-summary-document-attachment">
+                <div class="cx-account-summary-document-label">
+                  {{ 'orgAccountSummary.document.attachment' | cxTranslate }}
+                </div>
+                <div class="cx-account-summary-document-value">
+                  <a *ngIf="document.documentMedia"
+                     [href]="document.documentMedia"
+                     target="_blank"
+                     rel="noopener noreferrer">
+                    <cx-icon
+                      [type]="ICON_TYPE.FILE"
+                      class="cx-account-summary-document-attachment-icon"
+                      title="{{ 'orgAccountSummary.document.view' | cxTranslate }}">
+                    </cx-icon>
+                    <span class="cx-account-summary-document-attachment-text">
+                      {{ 'orgAccountSummary.document.view' | cxTranslate }}
```

Comment:
In the browser the "View" link that is underlined looks a bit weird. It looks like there is a space before the word view.